spec/routes/web/webhook/github_spec.rb

```diff
@@ -77,6 +77,13 @@
       expect(page.body).to eq({message: "GithubRunner[#{runner.ubid}] created"}.to_json)
     end
 
+    it "fails if not queued and runner_id is empty" do
+      send_webhook("workflow_job", workflow_job_payload(action: "waiting", runner_id: nil))
+
+      expect(page.status_code).to eq(200)
```

Comment:
Also, that's weird, a 200 with an error code?

Reply:
Actually, they are not failures. If we return a response other than 200, GitHub will display it as a failure, and our redelivery prog will attempt a retry.

GitHub sends all workflow_job events, even those without `runs-on: ubicloud` to us. I need to rename this function. These are not errors, simply no-ops.

https://github.com/ubicloud/ubicloud/blob/8d31f8d0562b64c480976c1a101e585bbe815a68/routes/web/webhook/github.rb#L84

Reply:
The renaming affects more files. I will address this in a separate PR.

Reply:
Sounds good, thanks for explaining